pkg/cli/admin/release/extract_tools.go

```diff
@@ -320,6 +320,14 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			missing.Insert(target.Mapping.Image)
 			continue
 		}
+		// make sure registry is the one we want, if not, replace it
+		specSplit := strings.Split(spec, "@sha256")
+		desired := strings.Split(o.From, "@sha256")
+		if specSplit[0] != desired[0] &&
+			!strings.Contains(desired[0], "registry.svc.ci.openshift.org") &&
+			!strings.Contains(desired[0], "quay.io/openshift-release-dev") {
```

Comment:
You can't hard code any of these, you'd need to take the address from user's input and make sure that it does not appear in the payload anywhere.

Reply:
I've updated this, please give it another pass,  thanks